src/test/resources/com/aws/greengrass/smbridge/config.yaml

```diff
@@ -1,7 +1,8 @@
 services:
-    aws.greengrass.MqttBridge:
-        parameters:
+    aws.greengrass.clientDevices.sm.bridge:
+        configuration:
             brokerServerUri: 'tcp://localhost:8883'
     main:
         dependencies:
-            - aws.greengrass.MqttBridge
\ No newline at end of file
+            - aws.greengrass.clientDevices.sm.bridge
```

Comment:
I'm not sure, since I haven't been able to get a jar deployed on my device, but I modeled this off the MQTT Bridge config.yaml, which declared a dependency on itself. Will investigate some more.

Reply:
Hmm... that is weird. If it is not used yet, lets not add the file without validation.